### DISCUSSION TITLE
GKE v11.25.4-gke.2100 on autopilot does not have managed prometheus enabled

### DISCUSSION THREAD

**Discussion by ryan-logik** [GoogleCloudPlatform/prometheus-engine#418] [Q&A] [answered]
I recently upgraded my autopilot cluster to v1.25.4-gke.2100 and noticed that the managed prometheus is still disabled for the cluster. I was able to deploy it / get it up and running when following the non-GKE Kubernetes cluster setup. 

Am I missing something, or was this supposed to be enabled by default after upgrade the v1.25+?

**Comment by pintohutch** [comment, marked as answer]
Hi @rypotocki - yes this is a known issue. We have a fix that should be rolling out soon.

In the meantime, you're able to enable it yourself using `gcloud container clusters update <CLUSTER_NAME> --enable-managed-prometheus`. Apologies for the confusion and inconvenience.
